lib/main_common.pm

```diff
@@ -1590,6 +1590,7 @@ sub load_extra_tests_prepare {
     loadtest "console/consoletest_setup";
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
     loadtest "console/hostname";
+    #loadtest "console/validate_lvm_encrypt" if (get_var('FULL_LVM_ENCRYPT')); 
```

Comment:
Not very a good idea to comment code, better to simply not add it, but if so I don't understand the purpose of this code...